test/module/irohad/ametsuchi/ametsuchi_test.cpp

```diff
@@ -24,16 +24,21 @@
 #include "ametsuchi/impl/postgres_block_query.hpp"
 #include "ametsuchi/impl/postgres_ordering_service_persistent_state.hpp"
 #include "ametsuchi/impl/postgres_wsv_query.hpp"
-#include "ametsuchi/impl/storage_impl.hpp"
+#include "ametsuchi/impl/wsv_restorer_impl.hpp"
 #include "ametsuchi/mutable_storage.hpp"
+#include "builders/protobuf/block.hpp"
+#include "builders/protobuf/transaction.hpp"
 #include "common/byteutils.hpp"
+#include "cryptography/hash_providers/sha3_256.hpp"
 #include "framework/test_subscriber.hpp"
 #include "model/account.hpp"
```

Comment:
Can't find usage of `model::Account` in the file, so the include seems unused. Remove the rest
Remove the rest of unused headers plz (may help `using namespace iroha::model` removal)

Reply:
needed for wsv->getAccount()

Reply:
But I removed a bunch of other includes.

Reply:
Exact type isn't used, it basically inherited from getAccount type